profiler/nsys_mine.py

```diff
@@ -258,6 +281,7 @@ def parse_args():
                 break
     assert skip_first_mainloop is False and min_start != 0
     timestep_time_in_ms = (float(max_end) - float(min_start)) * 1.0e3
+    print(f"Mining timespte between {min_start} and {max_end}")
```

Comment:
Potential typo, `timespte`, should be `timestep`?

Reply:
Nice catch